docs/usage.rst

```diff
@@ -643,8 +643,8 @@ The following example pictures a Pumped Hydroelectric Energy Storage (PHES). Bot
 
     solph.components.GenericStorage(
         label='PHES',
-        inputs={b_el: solph.flows.Flow(investment= solph.Investment(ep_costs=500))},
-        outputs={b_el: solph.flows.Flow(investment= solph.Investment(ep_costs=500)},
+        inputs={b_el: solph.flows.Flow(nominal_value= solph.Investment(ep_costs=500))},
+        outputs={b_el: solph.flows.Flow(nominal_value= solph.Investment(ep_costs=500)},
```

Comment:
Is there a specific reason for the spaces after the first `=` signs?

Reply:
Just typo, thanks.

```suggestion
        inputs={b_el: solph.flows.Flow(nominal_value=solph.Investment(ep_costs=500))},
        outputs={b_el: solph.flows.Flow(nominal_value=solph.Investment(ep_costs=500)},
```